project/script_IR.tcl

```diff
@@ -24,7 +31,14 @@ create_clock -period 240MHz -name slow_clock
 create_clock -period 360MHz -name fast_clock
 
 set nProc [exec nproc]
-csim_design -compiler gcc -mflags "-j$nProc"
```

Comment:
@aperloff Would it make sense to simply drop the `-compiler gcc` option altogether? In versions where the option is available, I think it defaults to gcc anyway, and in any case, I don't think we need a specific C++ compiler.

Reply:
I'd be happy just to from the `-compiler gcc` flag. I too don't see any downside. If you confirm, I will go ahead and remove the flag and the switch statement.

Reply:
I say go ahead and remove it.

Reply:
The `-compiler gcc` option is now removed.